src/main/webapp/app/shared/monaco-editor/monaco-editor.component.ts

```
@@ -161,33 +238,236 @@ export class MonacoEditorComponent implements OnInit, OnDestroy {
+
+        // Dispose selection change listeners
+        for (const listenerEntry of this.selectionChangeListeners) {
+            listenerEntry.disposable?.dispose();
+        }
+        this.selectionChangeListeners = [];
+
+        // Dispose snapshot model if present
+        this.disposeDiffSnapshotModel();
 
         // Clean up all per-model debounce timeouts
         this.textChangedEmitTimeouts.forEach((timeout) => clearTimeout(timeout));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)